### DISCUSSION TITLE
Please help! AIR Coin withdraw issue!

### DISCUSSION THREAD

**Discussion by Sanyny** [air-verse/air#193] [Q&A]
Hi,
I would like to know how do I go about swapping any BEP20 (AIR coin for example) from TrustWallet into a currency that allows me to transfer to my bank account and to cash out?
Please help. There's no information online about this.
Binance is not available in my area.
A step-by-step guidance will be much appreciated.

**Comment by danawoodman** [comment]
lol you realize this is not a crypto project right?
